Include/dictobject.h

```diff
@@ -31,6 +31,7 @@ PyAPI_FUNC(PyObject *) PyDict_Values(PyObject *mp);
 PyAPI_FUNC(PyObject *) PyDict_Items(PyObject *mp);
 PyAPI_FUNC(Py_ssize_t) PyDict_Size(PyObject *mp);
 PyAPI_FUNC(PyObject *) PyDict_Copy(PyObject *mp);
+PyAPI_FUNC(int) PyDict_IsLazyKey(PyObject *mp, PyObject *key);
```

Comment:
naming bikeshed!

how about `PyDict_IsLazyImport`?

the API we specified in the PEP (PR still pending, https://github.com/python/peps/pull/2613) is `importlib.is_lazy_import`, so I think this name would be consistent with it

I guess another nit is whether this should be a PyDict API or PyImport API

Reply:
Changed the naming.

About the position, I think it depends on which we care more in this API, the `dict` or the `key`.